editor/plugins/tiles/tiles_editor_plugin.h

```diff
@@ -126,14 +126,42 @@ class TilesEditorPlugin : public EditorPlugin {
 	void set_sorting_option(int p_option);
 	List<int> get_sorted_sources(const Ref<TileSet> p_tile_set) const;
 
-	virtual void edit(Object *p_object) override;
-	virtual bool handles(Object *p_object) const override;
-	virtual void make_visible(bool p_visible) override;
+	void edit(Object *p_object);
+	void make_visible(bool p_visible);
 
 	static void draw_selection_rect(CanvasItem *p_ci, const Rect2 &p_rect, const Color &p_color = Color(1.0, 1.0, 1.0));
 
 	TilesEditorPlugin();
 	~TilesEditorPlugin();
 };
 
+class TileMapHandlerPlugin : public EditorPlugin {
```

Comment:
Why not TileMapEditorPlugin, following the same naming scheme used for most EditorPlugins?

Reply:
Because it already exists.
Also handlers have a bit different structure than usual plugins; it's more like a dummy plugin that only forwards methods.